### PR TITLE
Use simplified pdo fallback query

### LINKED ISSUE #59: Use simplified simple query for PDO fallback

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #59

### CODE REVIEW

**Comment by pivnicek** [comment]
ok, @ondrejhlavacek , thanks for the check, I think it should be ok now.

**Comment by ondrejhlavacek** [comment]
How do you want to proceed? Deploy and run as a tag to test it? Or deploy it as a new version for all?

File: tests/Keboola/DbExtractor/ApplicationTest.php [hunk 1]
```diff
@@ -284,4 +284,35 @@ public function testPDOFallbackSimpleNoData(): void
...
+        $config['parameters']['tables'][0]['outputTable'] = 'in.c-main.pdo_test';
+
+        @unlink($this->dataDir . '/config.yml');
+        file_put_contents($this->dataDir . '/config.yml', Yaml::dump($config));
+
+        $process = new Process('php ' . $this->rootPath . '/src/run.php --data=' . $this->dataDir);
+        $process->setTimeout(300);
+        $process->run();
+
+        $this->assertEquals(0, $process->getExitCode());
```
Comment (ondrejhlavacek) [inline tests/Keboola/DbExtractor/ApplicationTest.php]:
```
$process->run();
$this->assertEquals(0, $process->getExitCode());
```
is euqal to 

`$process->mustRun();`

which throws an exception when there's an error

Comment (pivnicek) [reply tests/Keboola/DbExtractor/ApplicationTest.php]: ah, good tip!  That will need to change for the whole file though, maybe It's best to put it in an issue and update it in a separate PR.

**Comment by pivnicek** [comment]
As for proceeding, I see this as a bug fix and should be tagged and released.  We can try it on `master` tag first if you like, to confirm.

**Pull request merged**